test/util.py

```diff
@@ -295,7 +293,7 @@ def estimate_fee(function, inputs, address, abi_path, signature=None, nonce=None
     return extract_fee(output.stdout)
 
 
-def call(function, address, abi_path, inputs=None, max_fee=None):
+def call(function, address, abi_path, inputs=None):
```

Comment:
Making a starknet call no longer works with a max fee (e.g. if you try providing `--max_fee` to Starknet CLI's `starknet call`, it will complain about an unrecognized argument). Also we aren't using it anywhere in the code.